packages/marshal/src/marshal.js

```diff
@@ -333,11 +345,22 @@ export const makeMarshal = (
      * representation. Rather, fullRevive must validate that.
      *
      * @param {Encoding} rawTree must be hardened
+     * @param {string} key
```

Comment:
Likewise, I'd expect either
```suggestion
     * @param {Encoding} rawTree must be hardened
     * @param {string} [key]
```
or
```suggestion
     * @param {string} key
     * @param {Encoding} rawTree must be hardened
```